src/pages/studyView/StudyViewUtils.tsx

```diff
@@ -1397,4 +1398,13 @@ export function getClinicalDataCountWithColorByCategoryCounts(yesCount:number, n
         }
     }
     return getClinicalDataCountWithColorByClinicalDataCount(_.values(dataCountSet));
+}
+
+export function getStudyViewTabId(pathname:string) {
```

Comment:
We have the code available in results view(ResulstViewPageHelper) or patient view(no sure the file name). Could you just generalize that code and put to Util class in the shared module? @alisman do we have any Util class already in the portal related to path?

Reply:
Yes, they are pretty much the same. I can make a util function for them.